### ISSUE TITLE
Transpile new ruby into mruby syntax

### ISSUE THREAD

**Issue by grzegorz-jakubiak** [prism-rb/prism#20]
We might wanna have a look at this: https://github.com/ruby-next/ruby-next

**Comment by Widdershin** [comment]
Yeah @palkan reached out to me over email before the launch, I've been busy though so definitely need to look into it.

He also told me that mruby does actually support kwargs and null traversal, so we need to update the README.
